packages/editor/src/client/routes/articleList.tsx

```diff
@@ -194,7 +194,22 @@ export function ArticleList() {
                 if (rowData.pending) states.push(t('articles.overview.pending'))
                 if (rowData.published) states.push(t('articles.overview.published'))
 
-                return <div>{states.join(' / ')}</div>
+                return (
+                  <div
+                    style={{
+                      textAlign: 'center',
+                      borderRadius: '15px',
+                      backgroundColor: rowData.pending
+                        ? 'rgb(248, 222, 242)'
```

Comment:
Can you set the colors in a shared object or enum so it's easier to edit? There's a similar one already in the utilities file to copy or reuse.

Reply:
@penk85 thanks a lot for the note, fixed. Please review.